src/Footer.js

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { styled, alpha } from '@mui/material/styles';
+
+const FooterContainer = styled('div')(({ theme }) => ({
+    position: 'fixed',
+    left : 0,
+    bottom : 0,
+    color : 'white',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderRadius: theme.shape.borderRadius,
+    backgroundColor: alpha(theme.palette.common.black,1),
+    // '&:hover': {
```

Comment:
Ideally, don't commit any commented out code. Unless it's really essential to keep, it will then just cause confusion!